vars.tf

```diff
@@ -1,3 +1,13 @@
+variable "aws_region" {
```

Comment:
The region is not a concern of the resource module. Please remove

Reply:
removed